docs/pages/includes/tctl.mdx

```diff
@@ -1,4 +1,4 @@
-Make sure you can connect to Teleport. Log in to your cluster using `tsh`, then use `tctl`
+You can sign in to your Teleport cluster using `tsh` and can use `tctl`
```

Comment:
I think this loses the intent of this paragraph, which is to indicate that `tctl` is required for this guide, and to use it, a user needs to authenticate via `tsh login`.

Reply:
@ptgott 
I'm still trying to figure out how this `tsh login` followed by running `tctl` commands (that appears in pretty much every topic) should work. My experience doesn't jive with what it seems like the topic implies.

When I enroll a server as a resource, I can run `tsh login --proxy=proxy-address --user=admin-user`, authenticate as this user, then run `tsh` commands on that server. 

I can't run any `tctl` commands or `sudo tctl` commands on the enrolled server. I can only run `tctl` commands on the Auth service host. I've tried creating an identity file and using the `--auth-server` and `--identity` flags to run `tctl` commands on the enrolled server, but I haven't found the right combination of command-line options to make it work.

```
ubuntu@ip-192-168-13-57:~$ tctl status
ERROR: failed to open file for Teleport configuration: /etc/teleport.yaml. Ensure that you are running as a user with appropriate permissions.
open /etc/teleport.yaml: permission denied

ubuntu@ip-192-168-13-57:~$ sudo tctl status
ERROR: tctl must be either used on the auth server or provided with the identity file via --identity flag
```

Is it possible to run `tctl` commands on a non-auth-server? what's the magic incantation that allows it to happen? (I found this page https://goteleport.com/docs/reference/cli/#tctl, but I must be missing something that is obvious to others.)

Reply:
There are two ways to run tctl:

1. Local-mode: you execute tctl on the auth server directly. In this case, tctl looks for a config file at `/etc/teleport.yaml` to know how to communicate. (This was the original approach, but has fallen out of favor since cloud customers don't have direct access to their auth servers).
2. Remote mode: you run tctl on your workstation, and it connects to a teleport cluster running somewhere else. It does this by using the credentials that were generated from `tsh login` - this is why you have to run `tsh login` first.

> When I enroll a server as a resource, I can run tsh login --proxy=proxy-address --user=admin-user, authenticate as this user, then run tsh commands on that server.

This doesn't make any sense to me. When you enroll a server as a resource, you should run `tsh` on _your workstation` to access the resource with `tsh ssh`. It's rare that you would want to run tsh on the enrolled server. It's even more rare that you would want to run tctl on that server.

> I can't run any tctl commands or sudo tctl commands on the enrolled server.

What exactly are you trying to do? There aren't very many teleport use cases where one would want to run `tctl` on an enrolled server.

Reply:
I've been trying to do remote mode. Is the expectation that my workstation that connects to the cluster to get my credentials isn't (ever) a resource in the cluster? If so, that's probably what's tripping me up.

I was imagining the scenario was: 
I want to create an invitation token for server A, so I open a terminal session on server A, connect to the cluster remotely from that server, run the command to create the token on server A, and update the configuration on server A.
But it's more like my workstation (outside of the cluster) connects to the cluster remotely and connects to server A remotely. Is that how it works?

Reply:
> Is the expectation that my workstation that connects to the cluster to get my credentials isn't (ever) a resource in the cluster?

It's rare, but not impossible. We're actually adding a feature to Teleport Connect to make it easy to register your own workstation as a server in the cluster.

> But it's more like my workstation (outside of the cluster) connects to the cluster remotely and connects to server A remotely. Is that how it works?

Exactly. You would typically generate the token for server A from your workstation, and just copy the token over to server A. (If you're adding server A to a Teleport cluster, it's likely that you want multiple users to be able to access server A, so it doesn't make much sense for you to log in as yourself on server A where other users could grab your certificates)

Reply:
Thanks, @zmb3! That really helps and I appreciate your patience.